atspi-common/src/object_match.rs

```diff
@@ -57,9 +58,20 @@ pub struct ObjectMatchRule {
 //
 // https://gitlab.gnome.org/federico/at-spi2-core/-/commit/4885efedeef71e0df8210622771a0b1bb10e194d
 impl Type for ObjectMatchRule {
-	fn signature() -> Signature<'static> {
-		Signature::from_str_unchecked("(aiia{ss}iaiiasib)")
-	}
+	const SIGNATURE: &'static Signature = &Signature::static_structure(&[
+		&Signature::Array(Child::Static { child: &Signature::I32 }),
```

Comment:
I think it would be more readable to get the signatures from Rust's `Vec` and `HashMap` implementation of the `Type` trait here.